models/games.js

```diff
@@ -18,7 +18,12 @@ exports.allReviews = () => {
 
 exports.getReviewByRequest = (review_id) => {
   return db
-    .query ("SELECT * FROM reviews WHERE review_id = $1;", [review_id]
-    )
+    .query("SELECT * FROM reviews WHERE review_id = $1;", [review_id])
     .then((result) => result.rows[0]);
-};
\ No newline at end of file
+};
+
+exports.getCommentsByReviewId = (review_id) => {
+  return db
+    .query("SELECT * FROM comments WHERE review_id = $1;", [review_id])
+    .then((result) => result.rows);
+};
```

Comment:
In your models you may need a checkIfReviewExists function to differentiate between invalid reviews, and valid reviews with no comments. Both of these will return empty rows at this point.